packages/components/pie-toast/src/index.ts

```diff
@@ -147,7 +164,7 @@ export class PieToast extends RtlMixin(LitElement) implements ToastProps {
 
         return html`
             <pie-button
-                variant="ghost-inverse"
+                variant="${this.isVariantWarningAndIsStrong() ? 'ghost' : 'ghost-inverse'}"
```

Comment:
To future-proof this method, may want to call it something like `shouldInvertBtnVariant` and that way if design ever add other variants that also use the inverse icon variants, you wouldn't need to update the method name?

```suggestion
                variant="${this.isVariantWarningAndIsStrong() ? 'ghost' : 'ghost-inverse'}"
```

Reply:
That's a fair point, however the logic in the method itself wasn't changed so I just renamed to `shouldNotUseInverseBtnVariant`